### PR TITLE
Spliceai link - Fix #3256

### LINKED ISSUE #3256: Provide a simple explanation for the spliceAI numbers
For instance what a suspicious value is?

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by DN, CR

### CODE REVIEW

File: scout/server/blueprints/variant/templates/variant/variant_details.html [hunk 1]
```diff
@@ -253,7 +253,7 @@
           <span class="float-right">{{ variant.spidex_human }}</span>
         </li>
         <li class="list-group-item">
-          <a hfref="https://github.com/Illumina/SpliceAI">SpliceAI</a> DS max
+          <a href="https://doi.org/10.1016/j.cell.2018.12.015">SpliceAI</a> <a href="https://github.com/Illumina/SpliceAI">DS max</a>
```
Comment (northwestwitch) [inline scout/server/blueprints/variant/templates/variant/variant_details.html:256]: It's working fine! Consider to open the link in a new tab. But I see that the other links (sift, revel, etc) are opening on the same window so it's fine to keep it like this (or to make them all open on new tabs)

**Review by northwestwitch — APPROVED** [review]
Works perfectly!

File: scout/server/blueprints/variant/templates/variant/variant_details.html (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/variant_details.html:256]: hfref :)

**Comment by northwestwitch** [comment]
This can be merged, no?

**Comment by dnil** [comment]
> This can be merged, no?

Right, it was only that comment with opening in new windows. One moment..

**Pull request merged**